frontend/src/features/admin-form/common/queries.ts

```diff
@@ -99,7 +99,8 @@ export const useAdminFormCollaborators = (formId: string) => {
     // Collaborators is source of truth if it has already loaded.
     if (collaborators) {
       return collaborators.some(
-        (perms) => perms.write && perms.email === user.email,
+        (perms) =>
+          perms.write && perms.email.toLowerCase() === user.email.toLowerCase(),
       )
     }
     // Else use permissionList first
```

Comment:
should we add it to the lines below too?
```suggestion
    // Else use permissionList first
    return form.permissionList.some(
      (perms) => perms.write && perms.email.toLowerCase() === user.email.toLowerCase(),
    )
```

Reply:
good catch! 4e01e48